Code/monet_cyclegan/utils.py

```diff
@@ -284,4 +285,66 @@ def log_args(args: Namespace) -> None:
     """
 
     for arg, value in sorted(vars(args).items()):
-        logger.info(f'{arg}: {value}')
+        logging.info(f'{arg}: {value}')
+
+def ten_percent_for_test(jpg_train_file_path: str, jpg_test_file_path:str) -> None:
+    """This function parses a file path to a train dataset that contains jpg files, alters 10% of them and then adds them to the test data set
+
+    Args:
+        jpg_train_file_path: file path for the existing train file
+        jpg_test_file_path: file path for the existing test file
+    """
+    if os.path.exists(jpg_train_file_path)!=True:
+        return "Error: The train file path does not exist"
+    if os.path.exists(jpg_test_file_path)!=True:
+        return "Error: The test file path does not exist"
+    number_of_values_in_train_directory = len(os.listdir(jpg_train_file_path))
+    ten_percent_of_values = float(number_of_values_in_train_directory)/10.0
```

Comment:
We can use percent parameter instead of hardcoding 10.0

Reply:
note: using 10 percent requires the dataset to be at least size 10.

Reply:
I used ten percent because that was the number she mentioned. This function will be applied to the training datasets, so they will always be of size 10.